.ignore

```diff
@@ -1,3 +1,3 @@
-lib/
-dist/
-docs/
+packages/near-api-js/lib/
```

Comment:
What does `.ignore` actually ignore?

Reply:
It's for the [`ag` search script](https://github.com/near/near-api-js/commit/c7290650ecd851202e14bc56166f08852686a333). Don't want to change it in case it's being used by a dependency or some tooling unless we figure out what's dependent on it.